core/spec/lib/refinery/plugins_spec.rb

```diff
@@ -61,6 +97,11 @@ class Engine < ::Rails::Engine
         expect(subject.registered.in_menu.any?).to be_truthy
         expect(subject.registered.in_menu.all? { |p| !p.hide_from_menu }).to be_truthy
       end
+
+      it "orders by plugin_priority config" do
+        Core.config.plugin_priority = %w(my_ordered_plugin my_plugin my_other_ordered_plugin)
+        expect(subject.registered.in_menu.names.take(2)).to eq %w(my_ordered_plugin my_other_ordered_plugin)
```

Comment:
I think that to avoid the new routing errors we've just seen in CI, we are going to need to mock this.

Something like:

```ruby
allow(subject).to receive(:registered).and_return([
  double(name: "one"), 
  double(name: "two")
])
```

Or something like that.